### PR TITLE
dts: x86: intel: alder_lake: Added TGPIO instance

### PR DESCRIPTION
**Pull request by akanisetti**
Enable TGPIO support on alder lake platform.

Signed-off-by: Anisetti Avinash Krishna [anisetti.avinash.krishna@intel.com]

### CODE REVIEW

File: dts/x86/intel/alder_lake.dtsi [hunk 1]
```diff
@@ -430,6 +430,13 @@
 			status = "okay";
 		};
 
+		tgpio: tgpio@fe001200 {
+			compatible = "intel,timeaware-gpio";
+			reg = <0xfe001200 0x100>;
+			timer-clock = <19200000>;
+			max-pins = <2>;
+			status = "okay";
```
Comment (jhedberg) [inline dts/x86/intel/alder_lake.dtsi:438]: Nitpick: missing empty line after this

**Review by jhedberg — CHANGES_REQUESTED** [review]

File: dts/x86/intel/alder_lake.dtsi (hunk 1 above)
Comment (jhedberg) [reply dts/x86/intel/alder_lake.dtsi:438]: Clarification: after the node, ie have an empty line between each node definition.

Comment (akanisetti) [reply dts/x86/intel/alder_lake.dtsi:438]: Updated

**Review by jhedberg — APPROVED** [review]
@akanisetti looks good now, thanks

**Review by finikorg — APPROVED** [review]

**Pull request merged**